Assets/UI/Choosers/ResearchChooser_CQUI.lua

```diff
@@ -0,0 +1,59 @@
+include("ResearchChooser");
+
+-- ===========================================================================
+-- Cached Base Functions
+-- ===========================================================================
+BASE_CQUI_AddAvailableResearch = AddAvailableResearch;
+BASE_CQUI_OnOpenPanel = OnOpenPanel;
+
+-- ===========================================================================
+-- CQUI Members
+-- ===========================================================================
+local CQUI_AlwaysOpenTechTrees = false; --Ignores events calling for this to open when true
+local CQUI_ShowTechCivicRecommendations = false;
+
+-- ===========================================================================
+function CQUI_OnSettingsUpdate()
+    -- CQUI_AlwaysOpenTechTrees is a checkbox, so it has a value of true or false
+    CQUI_AlwaysOpenTechTrees = GameConfiguration.GetValue("CQUI_AlwaysOpenTechTrees");
+    -- CQUI_ShowTechCivicRecommendations is a ComboBox, so it has a vlaue of 0 or 1
```

Comment:
misspelling in the comment
```suggestion
    -- CQUI_ShowTechCivicRecommendations is a ComboBox, so it has a value of 0 or 1
```